app/helpers/feeds_helper.rb

```diff
@@ -0,0 +1,37 @@
+module FeedsHelper
+  def episode_offset_options
+    values = [300, 900, 3600, 21600, 43200, 86400]
+
+    I18n.t([:min_5, :min_15, :hr_1, :hr_6, :hr_12, :hr_24], scope: [:feeds, :helper, :episode_offset_options]).map.with_index { |label, i| [label, values[i]] }
```

Comment:
Hmm... can you not just have numbers (300, 900, 3600...) in en.yml?  It seems to slightly odd to have 2 versions of them in here.

Reply:
i will look at it again. i think i had trouble using integers as keys in the yml file, but i didn't investigate too deeply into how i was utilizing them here so i went with this weird intermediate way that i knew how to make work. but i will try mapping more directly and see if i was missing something in the implementation.

Reply:
Hrmm... maybe the yml file doesn't like integer keys?  Weird.  I would have thought something like this would "just work"-

```ruby
[96, 112, 128, 160, 192, 224, 256, 320].map do |val|
  [I18n.t(val), val]
end
```

Reply:
ok i think i got it. the keys have to be strings in the yml file. i can change this.